vm/src/sliceable.rs

```diff
@@ -335,10 +348,10 @@ pub enum SequenceIndex {
 }
 
 impl SequenceIndex {
-    fn try_from_object_for(
+    pub fn try_from_object_for(
+        owner_type: &'static str,
```

Comment:
I think the parameter order of this function shouldn't be changed.
We conventionally keep `vm`'s position as the first or the last. @coolreader18 is it?

Reply:
Yeah, I think it could probably be back to how it was before, `vm obj owner_type`